Remove unnecessary prefixes from CLI docs (#1254)
Tyrrrz/DiscordChatExporter: .docs/Using-the-CLI.md

## Step 1

> After extracting the `.zip`, open your terminal (Command Prompt or Powershell on **Windows**, and Terminal on **macOS** and **Linux**).

Reviewer: Removed the extra details because they introduce confusion between terminal/shell concepts. On Windows, users should use the Terminal app ideally as well, but that is not too important.

```suggestion
After extracting the `.zip` archive, open your preferred terminal.
```

## Step 3

Now we're ready to run the commands.

Type the following command in your terminal of choice, then press ENTER to run it. This will list all available subcommands and options.

```console
./DiscordChatExporter.Cli
```

> **Note**:

> > Command Prompt users must omit the `./` from the start of the command. The examples on this page use **Windows** file paths. **MacOS** and **Linux** users must use Unix file paths accordingly.

Reviewer: Simplified tone.

```suggestion
> On Windows, if you're using the default Command Prompt (`cmd`), omit the leading `./` at the start of the command.
```
